tests/std/tests/Dev11_0836436_get_time/test.cpp

```diff
@@ -702,3 +706,62 @@ void test_locale_chinese() {
     assert(read_date_locale(L"2020-\x5341\x4e8c\x6708-31", "zh-CN") == make_tuple(31, 11, 120));
     assert(read_date_locale(L"2020-\x0031\x0032\x6708-31", "zh-CN") == make_tuple(31, 11, 120));
 }
+
+void test_invalid_parameter_handler(const wchar_t* const expression, const wchar_t* const function,
+    const wchar_t* const file, const unsigned int line, const uintptr_t reserved) {
+    (void) expression;
+    (void) reserved;
+
+    // Stop test early. Without this,
+    static int num_called = 0;
+    if (num_called++ > 10) {
+        wprintf(L"Test Failed: Invalid parameter handler was called over 10 times by %s in %s:%u\n", function, file,
+            line); // These arguments are only populated in debug mode.
+        exit(1);

```

Comment:
Missing `#include<cstdlib>`.

Reply:
Thank you for review 👍  I'll make changes in some days